vignettes/visits_periods.Rmd

```diff
@@ -92,86 +92,212 @@ derive_vars_joined(
 
 # Period, Subperiod, and Phase Variables
 
-If periods, subperiods, or phases are used, the corresponding variables
+If periods, subperiods, or phases are used, in the simpler of ADaM applications the corresponding variables
 have to be consistent across all datasets. This can be achieved by
-defining the periods, subperiods, or phases once and then use this
+defining the periods, subperiods, or phases once and then using this
 definition for all datasets. The definition can be stored in ADSL or in
-a separate dataset. In the following examples, this separate dataset is
-called period reference dataset.
+a separate dataset. In this vignette's examples, this separate dataset is
+called period/phase reference dataset depending what data it contains.
 
-## Period Reference Dataset {#reference}
+Note that periods, subperiods, or phases *can* be defined differently across datasets (for instance, they may be different across safety and efficacy analyses) in which case the start/stop dates should be defined in the individual datasets, instead of in ADSL. However, this vignette will not cover this scenario.
 
-The period reference dataset contains one observation per subject and
-period, subperiod, or phase. For example:
+## A Note on Study Specific Code
+
+The sections below showcase the available tools in admiral to work with period, 
+subperiod and phase variables. However, at some point study specific code will 
+always be required. There are two options:
+
+-   Study specific code is used to first derive the variables `PxxSwSDT` and
+`PxxSwEDT` in ADSL. Then `create_period_dataset()` and
+`derive_vars_joined()` can be used to derive period/subperiod
+variables like `ASPER` or `ASPRSDT` in BDS and OCCDS datasets. See an example 
+dataset [here](#adsl_example).
+
+-   Study specific code is used to derive a dataset with one
+observation per patient, period, and subperiod (see [period
+reference dataset](#reference)). Then `derive_vars_period()` can be
+used to derive `PxxSwSDT` and `PxxSwEDT` in ADSL and
+`derive_vars_joined()` can be used to derive period/subperiod
+variables like `ASPER` or `ASPRSDT` in BDS and OCCDS datasets.
+
+It depends on the specific definition of the periods/subperiods which
+option works best. If the definition is based on other ADSL variables,
+the first option would work best. If the definition is based on
+vertically structured data like exposure data (EX dataset), the second
+option should be used. This vignette contains examples for both workflows.
+
+
+## The Period/Phase Reference Dataset {#reference}
+
+The admiral functions expect separate reference datasets for periods, subperiods,
```

Comment:
Did we want to use {} when admiral is written out without the word package after it?  I think this is our convention??

Reply:
yes agreed, updated.